src/modules/rolle/repo/rolle.repo.spec.ts

```diff
@@ -118,7 +121,68 @@ describe('RolleRepo', () => {
             expect(rolle).toBeNull();
         });
     });
+    describe('findRollenAuthorized', () => {
+        it('should return no rollen because there are none', async () => {
```

Comment:
Wir sollten vielleicht für dieses Repo auch Mocktests für solche Use-Cases schreiben, damit man nicht einfach Mülldaten in die DB schreiben wenn der ganze Test gemockt werden kann :)